app/build.gradle.kts

```diff
@@ -0,0 +1,69 @@
+plugins {
+    kotlin("android")
```

Comment:
Changes are looking good. Idea for future improvement - setup a convention plugin to configure stuff like Android for all modules.
https://docs.gradle.org/current/userguide/sharing_build_logic_between_subprojects.html